vars/sconsBuild.groovy

```diff
@@ -189,12 +188,12 @@ def call(Map config = [:]) {
     if (config['skip_clean']) {
         clean_cmd += "echo 'skipping scons -c'\n"
     } else {
-        def clean_files = "_build.external"
+        def clean_files = "install"
         if (config['clean']) {
             clean_files = config['clean']
         }
-        clean_files += ' install build {daos_m,daos,iof,cart-Linux}.conf'
-        clean_files += ' .sconsign{,-Linux}.dblite .sconf-temp{,-Linux}'
+        clean_files += ' build daos.conf'
+        clean_files += ' .sconsign.dblite .sconf_temp'
```

Comment:
Ditto.

Reply:
And the -Linux suffixes never existed in DAOS, only cart and IOF.

Reply:
And ditto.

Reply:
> And the -Linux suffixes never existed in DAOS, only cart and IOF.

Indeed.  But they did in cart which is still separate in our current (1.0.x) release, so if we want to have a goal of at lest being able to build that, we need to maintain this compatibility.  Which I see you have done.  But just justifying why.